docs/source/_developers/primer.md

```diff
@@ -0,0 +1,234 @@
+:::
+
+This style guide outlines the best practices and conventions for creating clear, consistent, high-quality documentation for our project. By following these guidelines, we can ensure a cohesive writing style and tone across all documentation.
+
+This style guide provides guidelines for writing clear, concise, and consistent documentation. It covers language, tone, formatting, and other aspects of documentation.
+
+##  Writing Style
+The writing style in documentation is critical for ensuring consistency and understanding. Here are some best practices:
+
+**Clarity**: Be as clear and straightforward as possible. Avoid ambiguity. Documentation is not a place for complex sentences and unusual words.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - The initiation of the replication process...
+  - To start the replication process...
+:::
+
+**Conciseness**: Keep sentences and paragraphs short. This makes text easier to read and understand.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - Once the process has been completed successfully, the next step is...
+  - After completing the process, next...
+:::
+
+**Active Voice**: Use the active voice whenever possible as it is more straightforward and engaging.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - The command can be executed by the user.
+  - The user can execute the command.
+:::
+
+**Address the Reader**: Use the second person ("you") to address the reader directly.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - One can see the results by... <br>
+  - You can see the results by...<br>
+:::
+
+- Avoid using jargon and slang, as they can be confusing.
+- Keep your writing easy to read by using short sentences and paragraphs.
+- If you need to explain complex concepts, break them into smaller, more manageable sections to help your reader understand your message.
+
+##  Formatting
+MyST allows for markdown to be used for our Spinx project (i.e., RTD).
+
+:::{admonition} Bookmark and Review-worth Reference
+Check out this [Markdown cheatsheet].
+:::
+
+
+Commonly used elements follow the [MyST-Typography].
+
+Directives allow us to insert and format various elements (e.g., images, callouts, etc.). We use `colon_fence` to replace the typical hyphens Learn more about  [Colon Fence].
+
+:::::{important}
+We use the `colon_fence` extension the markdown rendered by Myst parses three colons (i.e., *:*) to open and close blocks. ([about colon_fence]).
+
+::::{note}
+[Myst-Parser Documentation] includes several constructs we can use to enhance our pages. **Use elements with purpose, whether to call attention, reference related content, or warn.
```

Comment:
I agree using elements with purpose. It might be helpful to put guideline on when and where to use them. Like the discussion we had on Tabs. Where it would be appropriate to have for PyTorch/TensorFlow install but other sections it wouldn't be appropriate since we love the ability to link to those sections.

Reply:
I agree with @joseph-cooke. In fact, it would be quite helpful if we can put simple examples showing whatever you've put in text in action.

Reply:
Yes and yes, I agree that we should try and define the difference between what constitutes an "important", "note", or "tip", etc. If we want to be consistent in when we use them. 

I also wonder if on this page you can nest after the first paragraph and note, so that the second 'note' is nested within 'important' to better highlight nested blocks (or if this is not a great example of nested blocks we should illustrate one as we might actually use one)

Reply:
Oops just saw Manasvita's comment below, ditto mine above.

Reply:
There are a few points here.  

On the main (i.e., callouts/admonitions)
- There are several types of callouts which we could add specifications (the more defined, the better). @SerenaCaplins, Joseph is helping with OpenMPI page (which is a big one). Can you take the lead on this? See the bottom of this section, https://northeastern-university-rc-public-documentation--65.com.readthedocs.build/en/65/_developers/primer.html#formatting. The see also can have a note containing the specs, sub-bullets, a table, or even a subsubsection following that bullet list at the bottom (click that link to see a list of options, which was just added)

Reply:
The second on tabs: I added a subsection in Formatting for Tabs summarizing this discussion. Serena, it might work well to do the admonitions like this in a subsection right prior to this one.

Reply:
I do not entirely understand the third, @SerenaCaplins. If you can elaborate.

The part I understood is demoing the nesting of admonitions: this is done in the formatting section. Please let me know if I misunderstood.

Reply:
This was done. Please see updates for review.